### DISCUSSION TITLE
Agent has no effect

### DISCUSSION THREAD

**Discussion by MentosL** [apache/skywalking#10851] [Q&A] [answered]
After running my local project and configuring the agent, I did not see the current service on my skywalking (I requested an interface). What is this?

```java
-javaagent:path\skywalking-agent.jar=agent.service_name=backsever-service-local,collector.backend_service=ip:11800
```

![image](https://github.com/apache/skywalking/assets/33775732/145c7276-6e43-47c5-9c6a-1afbea06131f)

skywalking is blank
![image](https://github.com/apache/skywalking/assets/33775732/d68eb47a-6c65-4d69-9890-120f0609a0b0)

**Comment by MentosL** [comment]
![image](https://github.com/apache/skywalking/assets/33775732/a9dc8c43-e385-4252-bf69-52cdaca66276)

**Reply by Superskyyy** [reply]
In UI config it should be http://oap:port, you might have missed the scheme

**Reply by MentosL** [reply]
> http://oap:port you missed the scheme

sorry ，i don't understand you say，agent option need add scheme？ like ：
```java
-javaagent:path\skywalking-agent.jar=agent.service_name=backsever-service-local,collector.backend_service=http://ip:11800
```

**Reply by Superskyyy** [reply, marked as answer]
> > http://oap:port you missed the scheme
> 
> 
> 
> sorry ，i don't understand you say，agent option need add scheme？ like ：
> 
> ```java
> 
> -javaagent:path\skywalking-agent.jar=agent.service_name=backsever-service-local,collector.backend_service=http://ip:11800
> 
> ```

Its not java agents problem, please check II logs and it's very likely that you pointed to wrong oap address or oap failed to start.

**Reply by MentosL** [reply]
> > > http://oap:port you missed the scheme
> > 
> > 
> > sorry ，i don't understand you say，agent option need add scheme？ like ：
> > ```java
> > -javaagent:path\skywalking-agent.jar=agent.service_name=backsever-service-local,collector.backend_service=http://ip:11800
> > ```
> 
> Its not java agents problem, please check II logs and it's very likely that you pointed to wrong oap address or oap failed to start.

Thank you，yeah  you are right，i fix it in web-ui script。

**Comment by MentosL** [comment]
apache/skywalking-oap-server:8.9.1
apache/skywalking-ui:8.9.1
elasticsearch:7.9.0
